docs/app/Layouts/HomepageLayout.js

```diff
@@ -114,7 +114,7 @@ class MobileContainer extends Component {
     const { sidebarOpened } = this.state
 
     return (
-      <Responsive {...Responsive.onlyMobile}>
+      <Responsive {...Responsive.onlyMobile} minWidth={0}>
```

Comment:
This one also is specifically spreading a different value for minWidth.  Why is it that we need to override this to 0?

https://github.com/Semantic-Org/Semantic-UI-React/blob/8ccee0f75fb2a9802bbaf33394182dc97e8b5d2e/src/addons/Responsive/Responsive.js#L54

As above, if minWidth 0 is correct then we should not be spreading `onlyMobile` here, or, we need to update the values of `onlyMobile`.